### PR TITLE
Bookmark avatar entities

### PR DESCRIPTION
**Pull request by druiz17**
Added the ability to bookmark avatar entities and be able to switch between different bookmarks regardless of the avatar you are currently using

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6374/app_12191_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6374/HighFidelity-Beta-PR12191-3ba0eaa13d9906aaf1ef3787e17be4845242190f.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6374/HighFidelity-Beta-PR12191-3ba0eaa13d9906aaf1ef3787e17be4845242190f.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6394/app_12191_PR.apk

**Comment by druiz17** [comment]
Test plan
- wear some avatar entities and adjust them
- bookmark the avatar entities by clicking ```Avatar > Bookmark Avatar Entities.```
- add bookmarks for different types of avatar and avatar entities together
- select a saved bookmark for avatar entities from, ```Avatar > Avatar Entities Bookmarks```
- mark sure that the bookmark loads the correct avatar and avatar entities with the correct adjustment
- relaunch interface an preform the previous two tests

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6394/HighFidelity-Beta-PR12191-eadd3b48bd8d820cfc2bcd3a93f1be6170c2c25a.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6394/HighFidelity-Beta-PR12191-eadd3b48bd8d820cfc2bcd3a93f1be6170c2c25a.dmg

**Comment by sbennett77** [comment]
Currently Testing.

**Comment by sbennett77** [comment]
Passed.

@conklin94122

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6473/app_12191_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6474/app_12191_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6474/HighFidelity-Beta-PR12191-c71fa110dbda9a69ba225f8f29e88ecdefcd4525.exe

**Comment by themelissabrown** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6566/app_12191_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6566/HighFidelity-Beta-PR12191-c71fa110dbda9a69ba225f8f29e88ecdefcd4525.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6566/HighFidelity-Beta-PR12191-c71fa110dbda9a69ba225f8f29e88ecdefcd4525.exe

File: interface/src/AvatarEntitiesBookmarks.cpp [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+    for (auto it = _bookmarks.begin(); it != _bookmarks.end(); ++it) {
+        addBookmarkToMenu(menubar, it.key(), it.value());
+    }
+
+    Bookmarks::sortActions(menubar, _bookmarksMenu);
+}
+
+void AvatarEntitiesBookmarks::applyBookmarkedAvatarEntities() {
+    qDebug() << "AvatarEntitiesBookmarks::applyBookmarkedAvatarEntities";
```
Comment (ZappoMan) [inline interface/src/AvatarEntitiesBookmarks.cpp]: please delete log spam

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6677/app_12191_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6677/HighFidelity-Beta-PR12191-a637bac2fee71c2e4a78f885462bea5c568b81ab.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6677/HighFidelity-Beta-PR12191-a637bac2fee71c2e4a78f885462bea5c568b81ab.dmg

**Pull request merged**